plugins/scalac/src/main/scala/scala/macros/internal/plugins/scalac/quasiquotes/Macros.scala

```diff
@@ -92,7 +92,7 @@ class Macros(val c: Context) {
 
   private def parseSkeleton(input: m.Input, mode: Mode): m.Tree = {
     try {
-      val dialect = m.Dialect(
+      val dialect = m.dialects.Scala212.copy(
```

Comment:
It's weird that backend needs to deal with quasiquotes -- quasiquotes can and should be handled in the abstract part based on extractors and constructors -- that's so-called `platform-independent quasiquotes`, one of the main hypothesis of the new approach. But that's an issue for another PR.

Reply:
Looking forward to that!

Reply:
I agree @liufengyun. That is an issue for another PR. I had to change this line here since we recently added new fields to Dialect.